bubble-jam/Source/Presentation/Services/DatetimeService/DatetimeService.swift

```diff
@@ -23,16 +23,46 @@ struct DatetimeService: DateServicing {
         return dateComponent
     }()
     
-    func getCurrentDate() -> Date {
-        return Date.now
+    func getCurrentDate() async -> Date? {
+        let url = URL(string: "https://www.google.com")!
+        let request = URLRequest(url: url)
+        
+        do {
+            let (_, response) = try await URLSession.shared.data(for: request)
+            let httpRespone = response as? HTTPURLResponse
+            if let content = httpRespone!.allHeaderFields["Date"] as? String {
+                print(content)
+                let formatter = DateFormatter()
+                formatter.dateFormat = "EEE, dd MMM yyyy HH:mm:ssz"
+                formatter.timeZone = TimeZone.current
+                formatter.locale = Locale.current
+                let serverDate = utcToLocal(dateStr: content)
+                return serverDate
+            } else {
+                raise(1)
+            }
+        } catch {
+            print("deu merda")
```

Comment:
já que a gente não tá pegando o erro. Talvez fosse uma boa usar o `try? await URLSession.shared.data(for: request)` e só verificar se o valor é nulo ou não. Ao invés de usar o block try/catch